docs/proposals/performance.md

```diff
@@ -0,0 +1,21 @@
+# Post-1.0 performance/scalability improvement ideas
+
+This document is a collection of ideas for improving Kubernetes
+performance/scalability post-1.0. Feel free to add to it. Please
+link to relevant issues and assign an owner. Please only add things
+that will actually improve performance.
+
+## Performance/scalability goal
+
+The next performance/scalability goal is 1000 nodes with 30 pods/node. We may wish to revisit the 30 pods/node part if we find that this is far above what real users need.
+
+## Ideas
+
```

Comment:
might want to mention something about evaluating a multi member etcd cluster and/or sharding?